feature-libs/organization/administration/components/shared/item-active.directive.ts

```diff
@@ -0,0 +1,52 @@
+import { Directive, Input, OnDestroy, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { GlobalMessageType } from '@spartacus/core';
+import { tap } from 'rxjs/operators';
+import { OrganizationItemService } from './organization-item.service';
+import { MessageService } from './organization-message/services/message.service';
+import { BaseItem } from './organization.model';
+
+@Directive({
+  selector: '[cxOrgItemActive]',
+})
+export class ItemActiveDirective<T = BaseItem> implements OnInit, OnDestroy {
+  protected subscription;
+
+  @Input('cxOrgItemActive') form: FormGroup;
+
+  constructor(
+    protected itemService: OrganizationItemService<T>,
+    protected messageService: MessageService,
+    protected activatedRoute: ActivatedRoute
+  ) {}
+
+  ngOnInit() {
+    if (this.activatedRoute.snapshot.routeConfig.path === 'edit') {
```

Comment:
This is unfortunate, as we bind to the route.

Reply:
This was an additional conditional which is not needed, since we check if we have the item or not. Removed